platform/ui/src/views/StudyList/StudyList.js

```diff
@@ -83,14 +95,174 @@ const filtersValues = {
   resultsPerPage: 25,
 };
 
-const StudyList = ({ studies, perPage }) => {
-  const studiesData = studies.slice(0, perPage);
+const StudyList = () => {
+  const studies = utils.getMockedStudies();
   const numOfStudies = studies.length;
-  const isEmptyStudies = numOfStudies === 0;
+  const [expandedRows, setExpandedRows] = useState([]);
+
+  const tableDataSource = studies.map((study, key) => {
+    const rowKey = key + 1;
+    const isExpanded = expandedRows.some((k) => k === rowKey);
+    const {
+      AccessionNumber,
+      Modalities,
+      Instances,
+      StudyDescription,
+      PatientId,
+      PatientName,
+      StudyDate,
+      series,
```

Comment:
Only issue is if any of these are mapped values for the "view", we probably don't want them using these normalized names and naming convention. We may need to break from the rules the data services follow here. Fine for now. Just making a note as this could cause confusion when we integrate.